apps/bookmark/test/marionette/client_options.js

```diff
@@ -5,8 +5,6 @@ module.exports = {
     'dom.w3c_touch_events.enabled': 1
   },
   settings: {
-    'ftu.manifestURL': null,
-    'lockscreen.enabled': false,
     'keyboard.ftu.enabled': false,
```

Comment:
This patch is getting a bit big and hard to maintain. I would prefer to do it in a follow-up.